packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -41,7 +44,10 @@ export function PublishArticlePanel({
         {pendingPublishDate && (
           <Message
             type="warning"
-            description={t('articleEditor.panels.articlePending', {pendingPublishDate})}
+            description={t('articleEditor.panels.articlePending', {
```

Comment:
With the formatting options in i18next it should be possible to pass just one date object and do the formatting in the translation files.